src/modules/CC1101/CC1101.cpp

```diff
@@ -89,7 +89,8 @@ int16_t CC1101::begin(float freq, float br, float freqDev, float rxBw, int8_t po
   RADIOLIB_ASSERT(state);
 
   // set default sync word
-  state = setSyncWord(0x12, 0xAD, 0, false);
+  uint8_t sw[2] = RADIOLIB_CC1101_DEFAULT_SW;
```

Comment:
Either keep it as `uint8_t sw[]` or use the sync word length macro (I think the second option is better, to be more explicit).